### DISCUSSION TITLE
[Question][Hw1] Can we use frequency for digraphs/trigraphs?

### DISCUSSION THREAD

**Discussion by m-spitfire** [prosyslab-classroom/cs348-information-security#661] [Q&A]
Name: Murad Bashirov

Hello! In the implementation of HW1, can we use frequencies of digraphs(2 consecutive letters)/trigraphs apart from the given single letter frequency? Thanks.

**Comment by sujin0529** [comment]
Hi,
You use the single letter frequency basically.
But, if you need to use the frequencies of digraphs and trigraphs, you can use the frequencies.

Thanks.
